src/displayapp/screens/Calendar.cpp

```diff
@@ -9,7 +9,7 @@ Calendar::Calendar(DisplayApp* app, Pinetime::Controllers::Battery& batteryContr
 
     // Status bar clock and battery
     label_time = lv_label_create(lv_scr_act(), nullptr);
-    lv_label_set_text_fmt(label_time, "%02i:%02i", dateTimeController.Hours(), dateTimeController.Minutes());
+    lv_label_set_text_fmt(label_time, dateTimeController.FormattedTime().c_str());
```

Comment:
I think the `_fmt` function can be replaced with the normal `set_text` function here
```suggestion
    lv_label_set_text(label_time, dateTimeController.FormattedTime().c_str());
```